nnvm/python/nnvm/frontend/tensorflow.py

```diff
@@ -648,6 +648,13 @@ def _impl(inputs, attr, params):
             ignores=['Tpaddings'],)(new_inputs, attr)
     return _impl
 
+def _transpose():
+    def _impl(inputs, attr, params):
+        # Get its param which is a Const op
+        param_name = inputs[1].list_output_names()[0]
```

Comment:
[Ref. ](https://www.tensorflow.org/api_docs/python/tf/transpose) 

perm is optional argument please handle if not found in params.

Reply:
Also assert if conjugate is true in params as we don't support.

Reply:
If perm is not specified, tensorflow graph will introduce 'Rank' and 'Range' which I just ignored here, is that OK for you?

If conjugate is true, the actual tensorflow op will be 'ConjugateTranspose', so we don't need to handle it in conversion of 'Transpose'